edsnlp/pipelines/misc/dates/dates.py

```diff
@@ -40,7 +40,7 @@ class Dates(BaseComponent):
         (eg `pendant trois mois`).
     false_positive : Union[List[str], str]
         List of regular expressions for false positive (eg phone numbers, etc).
-    on_ents_only : Union[bool, str, List[str]]
+    on_ents_only : Union[bool, str, List[str], Set[str]]
```

Comment:
Would a broader type work here ?
```suggestion
    on_ents_only : Union[bool, str, Iterable[str]]
```